cob_hardware_config/raw3-1/config/arm_controller.yaml

```diff
@@ -32,4 +32,78 @@ joint_trajectory_controller:
   state_publish_rate:  25
   action_monitor_rate: 10
 
+## position controller
+joint_group_position_controller:
+  type: position_controllers/JointGroupPositionController
+  joints:
+     - arm_shoulder_pan_joint
+     - arm_shoulder_lift_joint
+     - arm_elbow_joint
+     - arm_wrist_1_joint
+     - arm_wrist_2_joint
+     - arm_wrist_3_joint
+  required_drive_mode: 1
+
+arm_1_joint_position_controller:
+  type: position_controllers/JointPositionController
+  joint: arm_shoulder_pan_joint
+  required_drive_mode: 1
+arm_2_joint_position_controller:
+  type: position_controllers/JointPositionController
+  joint: arm_shoulder_lift_joint
+  required_drive_mode: 1
+arm_3_joint_position_controller:
+  type: position_controllers/JointPositionController
+  joint: arm_elbow_joint
+  required_drive_mode: 1
+arm_4_joint_position_controller:
+  type: position_controllers/JointPositionController
+  joint: arm_wrist_1_joint
+  required_drive_mode: 1
+arm_5_joint_position_controller:
+  type: position_controllers/JointPositionController
+  joint: arm_wrist_2_joint
+  required_drive_mode: 1
+arm_6_joint_position_controller:
+  type: position_controllers/JointPositionController
+  joint: arm_wrist_3_joint
+  required_drive_mode: 1
+
+## velocity controller
+joint_group_velocity_controller:
+  type: velocity_controllers/JointGroupVelocityController
+  joints:
+     - arm_shoulder_pan_joint
+     - arm_shoulder_lift_joint
+     - arm_elbow_joint
+     - arm_wrist_1_joint
+     - arm_wrist_2_joint
+     - arm_wrist_3_joint
+  required_drive_mode: 2
+
+arm_1_joint_velocity_controller:
+  type: velocity_controllers/JointVelocityController
```

Comment:
I tested it at it works. I will keep the both options for the drivers for the ones that don't have the new driver... The configuration files are inside the driver package. I don't think it is  a good idea to change that...